core/controllers/simulationcontroller/simulation_controller.go

```diff
@@ -45,6 +46,37 @@ func New(simulator simulation.Simulator) *SimulationController {
 func (c *SimulationController) Attach(router *mux.Router) {
 	r := utils.CreateSubrouter(router, c.prefix)
 	r.Methods("POST").Path("/simulate").HandlerFunc(c.DoSimulation)
+	r.Methods("POST").Path("/simulation/start").HandlerFunc(c.StartApi)
+
+}
+
+func (c *SimulationController) StartApi(rw http.ResponseWriter, r *http.Request) {
```

Comment:
Can we rename this handler to `StartSimulation`? The current name is bit ambiguous

Reply:
@amneet270 Resolve this ASAP so we can close this issue + branch is out of date